s/src/redux/configureStores.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please don't disable eslint rules. There are here to help us write best code possible.